activesupport/lib/active_support/test_case.rb

```diff
@@ -45,6 +45,41 @@ def test_order
 
         test_order
       end
+
+      def run(reporter, options = {})
+        if filter = line_filter_for(options[:patterns] || [])
```

Comment:
We need to check which runnable implements a test - we don't have access to the runnable in `rails_minitest_plugin`.

Also adding it to Runner would make Active Support depend on Railties - this way stays clear of that.